scripts/parse-line-break.js

```diff
@@ -11,45 +12,41 @@ const findCanonicalName = function(shortName) {
 	return canonicalName;
 };
 
-const handled = new Set();
```

Comment:
The `handled` set are shared different Unicode versions, so previously we generated incorrect `Line_Break/Unknown` data because there is certain character, e.g. U+FE10, that is in LineBreak.txt as of Unicode 15.1.0 but removed as of Unicode 16.0.

Reply:
Great catch!

Reply:
Thanks. Could you cut a new patch release?

Reply:
Done in v1.6.3